github4s/src/test/scala/github4s/integration/GHActivitiesSpec.scala

```diff
@@ -19,36 +19,41 @@ package github4s.integration
 import cats.effect.IO
 import github4s.Github
 import github4s.domain._
-
 import github4s.utils.{BaseIntegrationSpec, Integration}
 
 trait GHActivitiesSpec extends BaseIntegrationSpec {
 
   "Activity >> Set a thread subscription" should "return expected response when a valid thread id is provided" taggedAs Integration in {
-    val response =
-      Github[IO](accessToken).activities
-        .setThreadSub(validThreadId, true, false, headerUserAgent)
-        .unsafeRunSync()
+    val response = client
+      .use { client =>
```

Comment:
Maybe use a short name like `c` here, or rename the `client` in `BaseIntegrationSpec` to `clientResource` or something like, to avoid shadowing?